### PR TITLE
API-3418: Update and standardize Benefits APIs descriptions

### PR DESCRIPTION
**Pull request by SlakrHakr**
### Description
https://vajira.max.gov/browse/API-3418
Verbiage on general benefits api information was old and outdated. This strives to update it.

### Process
Tested locally to ensure new verbiage shows up where expected.

### Requested feedback
That I didn't miss any specific places where similar updates to this verbiage might need to take place.

### CODE REVIEW

**Comment by rtravitz** [comment]
@SlakrHakr You'll need to update the visual regression tests. Instructions are here: https://github.com/department-of-veterans-affairs/developer-portal/blob/master/docs/testing.md#visual-regression-testing

**Review by jw81 — APPROVED** [review]

**Review by charleystran — APPROVED** [review]
LGTM

File: src/content/apiDocs/benefits/index.ts [hunk 1]
```diff
@@ -9,9 +9,9 @@ import ClaimsAttributesReleaseNotes from './claimsAttributesReleaseNotes.mdx';
 const benefitsContent: APICategoryContent = {
   intro: BenefitsIntro,
   overview: BenefitsOverview,
-  placardText: 'Submit benefits-related PDFs',
+  placardText: 'Build tools to electronically submit and track the status of benefit claims.',
   shortDescription:
-    'Enables approved organizations to submit benefits-related PDFs and access information on a Veteran’s behalf.',
+    'Enables Veterans and approved organizations to electronically submit and track the status of benefit claims.',
```
Comment (mjlumetta) [inline src/content/apiDocs/benefits/index.ts:14]: what's the difference between these two fields, from your perspective?

**Review by mjlumetta — APPROVED** [review]
LGTM ✨

File: src/content/apiDocs/benefits/index.ts (hunk 1 above)
Comment (SlakrHakr) [reply src/content/apiDocs/benefits/index.ts:14]: I think the main goal with these changes was to remove the "submit pdf's" language in favor of "electronically submit" verbiage. The rest of this is probably just to help people understand that its not just Veteran representatives that can use this API, but Veteran's themselves if they so choose.

**Pull request merged**